src/resources/taskData.json

```diff
@@ -4706,7 +4706,7 @@
         {
             "id": "965",
             "name": "200 Slayer Tasks",
-            "description": "NOTE: This task is a duplicate of \u201cComplete 200 Slayer Tasks\u201d and is currently bugged. Most players are unable to complete it, but some are, so it will remain on this list until Jagex fixes it.",
```

Comment:
I'm not sure why values were re-encoded. This was generated by running parseTaskSpreadsheet.py.